src/pages/rest/modules/import/index.md

```diff
@@ -14,6 +14,8 @@ The `POST /rest/<store_view_code>/V1/import/csv` endpoint allows you to use REST
 
 Your source data must be in the order and format Commerce expects. To understand the requirements of each import entity, go to the Import page in the Admin, select an entity type from the drop-down menu, then click **Download Sample File**. The endpoint accepts CSV data in base64 encoded format. Because you control the conversion of a file or data into base64, you do not need to escape special characters before the import.
 
+Before encoding the data in base64,  you can optionally use gzip compression to compress your data.  This is useful when importing large CSV files that may otherwise run into HTTP request size limitations of HTTP server or PHP configuration.
```

Comment:
Maybe I should have just said "gzip" instead of "gzip compression" since its redundant,  but not sure if everyone knows about gzip.